src/CoreFoundation/CFString.cs

```diff
@@ -146,7 +146,8 @@ public static NativeHandle CreateNative (string? value)
 			if (value is null)
 				return NativeHandle.Zero;
 
-			return CFStringCreateWithCharacters (IntPtr.Zero, value, value.Length);
+			using var valuePtr = new TransientString (value);
```

Comment:
`CFStringCreateWithCharacters` takes a Unicode string:

```suggestion
			using var valuePtr = new TransientString (value, TransientString.Encoding.Unicode);
```